test/jdk/java/foreign/handles/lookup_module/handle/lookup/MethodHandleLookup.java

```diff
@@ -117,6 +126,48 @@ public void testRestrictedHandles(MethodHandle handle, String testName) throws T
                     { MethodHandles.lookup().findVirtual(MemoryAddress.class, "set",
                             MethodType.methodType(void.class, ValueLayout.OfAddress.class, long.class, Addressable.class)),
                             "MemoryAddress::set/address" },
+                    { MethodHandles.lookup().findVirtual(MemoryAddress.class, "getAtIndex",
```

Comment:
I realized this class was not testing _all_ restricted methods - so I have added the missing test cases.